app/views/static_pages/tools_resources.html.erb

```diff
@@ -1 +1,22 @@
-<h1> tools and resources </h1>
+<h1> Tools/Resources </h1>
+<div class="playbook">
+  <h3>TechHire Playbook</h3>
+  <%= link_to 'Download PDF', download_path, class: "button" %>
+</div>
+<div class="case-studies">
+  <div class="business-case">
+    <h3>The Case for Businesses</h3>
+  </div>
+  <div class="mastercard-case">
+    <h3>MasterCard Case Study</h3>
+    <button name="playbook">Download the MasterCard Case Study</button>
+  </div>
+  <div class="saic-case">
+    <h3>SAIC Case Study</h3>
+    <button name="playbook">Download the SAIC Case Study</button>
+  </div>
+</div>
+<div class="fact-sheet">
+  <h3>Fact Sheet</h3>
+  <button name="playbook">Download the Factsheet</button>
+</div>
```

Comment:
I'm curious how it will handle multiple download files? Are those linked up here? I only see one link for download...do the buttons download too?